api/server/repository.js

```diff
@@ -1,7 +1,16 @@
 import GithubAPI from 'github-api'
 import _ from 'lodash'
+import Bluebird from 'bluebird'
```

Comment:
Since you started using Bluebird, maybe it's better to replace all the native Promise with Bluebird in this file?